src/main/scala/uncore/ahb/Test.scala

```diff
@@ -25,7 +25,7 @@ class AHBFuzzBridge()(implicit p: Parameters) extends LazyModule
   val gpio  = LazyModule(new RRTest0(0x100))
 
   model.node := fuzz.node
-  xbar.node := TLToAHB()(model.node)
+  xbar.node := TLToAHB()(TLDelayer(0.1)(model.node))
```

Comment:
That's the percentage chance that it delays a message. You probably want it to be specific to the test. Too many delays + too high => slow test.